src/__tests__/directives.test.ts

```diff
@@ -1383,4 +1383,54 @@ describe("Execute: handles directives", () => {
       expect(mockResolver).toHaveBeenCalled();
     });
   });
+
+  describe("default values in include / skip directive's if", () => {
+    test("default value true in include", () => {
+      const result = executeTestQuery(`
+        query myQuery($inc: Boolean = true) {
+          a @include(if: $inc)
+        }
+      `);
+
+      expect(result).toEqual({
+        data: { a: "a" }
+      });
+    });
+
+    test("default value false in include", () => {
+      const result = executeTestQuery(`
+        query myQuery($inc: Boolean = false) {
+          a @include(if: $inc)
+        }
+      `);
+
+      expect(result).toEqual({
+        data: {}
+      });
+    });
+
+    test("default value true in skip", () => {
+      const result = executeTestQuery(`
+        query myQuery($skip: Boolean = true) {
+          a @skip(if: $skip)
+        }
+      `);
+
+      expect(result).toEqual({
+        data: {}
+      });
+    });
+
+    test("default value false in skip", () => {
```

Comment:
maybe also add test when there is a variable passed to the query and query has also default?

Reply:
like 

```
query bla(a: Boolean = True) { ... }

variables(a: False)
```

Reply:
those tests are already present in variables.test.ts